src/rdf4cpp/rdf/storage/tuple/DefaultSolutionSequenceBackend.hpp

```diff
@@ -2,17 +2,20 @@
 #define RDF4CPP_DEFAULTSOLUTIONSEQUENCEBACKEND_HPP
```

Comment:
Missing `#ifndef`

Reply:
it is actually there, just the part that github shows does start at line 2

Reply:
ups, sorry about that